docs/en/verifying-signatures.wml

```diff
@@ -9,199 +9,380 @@
+    <p>
+    Some software sites list
+    <a href="https://en.wikipedia.org/wiki/Cryptographic_hash_function">sha1
     hashes</a> alongside the software on their website, so users can
     verify that they downloaded the file without any errors. These
     "checksums" help you answer the question "Did I download this file
     correctly from whoever sent it to me?" They do a good job at making
     sure you didn't have any random errors in your download, but they
     don't help you figure out whether you were downloading it from the
     attacker. The better question to answer is: "Is this file that I
-    just downloaded the file that Tor intended me to get?"</p>
-
-    <h3>Where do I get the signatures and the keys that made them?</h3>
+    just downloaded the file that Tor intended me to get?"
+    </p>
+    <a class="nav" href="#TOC" title="go up">&uarr;</a>
+  </article>
+ </div>
+
+<!-- Import Tor's signing key -->
+ <div>
+  <input id="ac-2" name="accordion-2" type="checkbox" checked />
+  <label for="ac-2">
+    <a class="nav" title="link here" href="#ImportKey">&#9668;</a>
+    <h3><a name="ImportKey">Step 1: Import Tor's signing key</a></h3>
     <hr>
-    <p>Each file on <a href="<page download/download>">our download
+    <p>
+    Each file on <a href="<page download/download>">our download
     page</a> is accompanied by a file with the same name as the
     package and the extension ".asc". These .asc files are GPG
     signatures. They allow you to verify the file you've downloaded
     is exactly the one that we intended you to get. For example,
     torbrowser-install-<version-torbrowserbundle>_en-US.exe is accompanied by
     torbrowser-install-<version-torbrowserbundle>_en-US.exe.asc. For a list
-    of which developer signs which package, see our <a href="<page docs/signing-keys>">signing keys</a> page.</p>
-
-    <p>We now show how you can verify the downloaded file's digital signature on
-    different operating systems. Please notice that a signature is dated the moment
-    the package has been signed. Therefore every time a new file is uploaded a new
-    signature is generated with a different date. As long as you have verified the
-    signature you should not worry that the reported date may vary.
+    of which developer signs which package, see our
+    <a href="<page docs/signing-keys>">signing keys</a> page.
     </p>
-
-    <h3>Windows</h3>
-    <hr>
-    <p>First of all you need to have GnuPG installed before you can verify signatures.
-    Download it from <a href="https://gpg4win.org/download.html">https://gpg4win.org/download.html</a>.</p>
-    <p>Once it's installed, use GnuPG to import the key that signed your
+  </label>
+  <article class="ac-box">
+
+   <p><span class="hint">Choose your operating system:</span>
+   <label for="ac-2-1"><a name="KeyWindows">&#9658; Windows</a></label>
+   <label for="ac-2-2"><a name="KeyMacOS">&#9658; Mac OS X</a></label>
```

Comment:
Apple calls it "macOS", but it might be ok to use "Mac OS X" if that name is easier for users.